installer/bundle/create_bundle.sh

```diff
@@ -155,8 +159,13 @@ OUTPUT_DIR=`(cd $2; pwd -P)`
 
 cd $INTERMEDIATE_DIR
 
+if [ ! -f $SOURCE_DIR/bundle_skel_${PLATFORM_TYPE}.sh ]
+then
+    echo "shell bundle skeleton file bundle_skel_${PLATFORM_TYPE}.sh does not exist in ${SOURCE_DIR}"
+    exit 1
+fi
 # Fetch the bundle skeleton file
-cp $SOURCE_DIR/bundle_skel.sh .
+cp $SOURCE_DIR/bundle_skel_${PLATFORM_TYPE}.sh ./bundle_skel.sh
```

Comment:
Might be with pre-flighting this with a meaningful exit code.  Something like:

```
if [ ! -f $SOURCE_DIR/bundle_skel_${PLATFORM_TYPE}.sh]
then
    echo "Can't find source bundle file $SOURCE_DIR/bundle_skel_${PLATFORM_TYPE}.sh ..."
    exit 1
fi
```

Then do the cp operation at 168.

The concern is that if something was changed, it might be hard to figure out exactly why the create_bundle script is unhappy ...